docs/graphman-graphql-api.md

```diff
@@ -0,0 +1,213 @@
+# Graphman GraphQL API
+
+When starting the `graph-node` optionally, a graphman GraphQL server can be started, which makes it possible to execute
+graphman commands via GraphQL.
+
+To start the graphman GraphQL server, the `GRAPHMAN_SERVER_AUTH_TOKEN` environment variable should be set. It is
+required that the authentication token be exactly 64 characters long. This is equivalent to an SHA-256 hash represented
```

Comment:
Do we need to require that? Why can't we leave it up to the person running `graph-node` how big/secure they want to make their token?

Reply:
Removed that requirement